miniti-client/src/app/tickets.service.ts

```diff
@@ -0,0 +1,30 @@
+import {Injectable} from '@angular/core';
+import {Ticket} from './ticket';
+import * as titleData from './dummy-titles.json';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TicketsService {
+
+  constructor() {
+  }
+
+  getTickets() {
+    const dummyTickets = [];
+
+    for (let i = 0; i < 20; i++) {
+      dummyTickets.push(new Ticket(i, _getDummyTicketTitle()));
+    }
+
+    return dummyTickets;
+
+    function _getDummyTicketTitle() {
```

Comment:
Really? I am a huge fan of inner functions :) They help me structure a function into better readable tinier bits without polluting the class because there is only this exactly one place where I need it. Why you do not like them?

Reply:
For me they made the code harder to read (maybe i'm not used to use them). As far as I read, there is no agreement about this topic in the community, (looks like a never ending war). So, no objections for me then.